test/functional/bundleadd/add-overwrite-files.bats

```diff
@@ -35,7 +36,7 @@ test_setup() {
 	# already exists in the target system, it should replace the file if the
 	# file is different
 
-	run sudo sh -c "$SWUPD bundle-add $SWUPD_OPTS test-bundle2"
+	run sudo sh -c "$SWUPD bundle-add $SWUPD_OPTS test-bundle2 --debug"
```

Comment:
is this --debug a leftover or you actually wanted that?

Reply:
leftover. I'll remove it